app/components/private-cloud/cost-chart-config.ts

```diff
@@ -79,14 +68,43 @@ export function getQuarterlyCostChartConfig({
   ];
 
   if (!forecast) {
-    dynamicChartData.pop();
-    dynamicChartData.pop();
+    datasets.pop();
+    datasets.pop();
```

Comment:
For my own understanding, why are there 2 pop() calls here?

Reply:
Will there always be only 2 charts? I'm wondering if it's possible to use the number of items in the list and pop the number of times based on that? Rather than hardcoding the 2 pops? That way if there is ever a case where there is not 2 charts, the code will not need to be updated to add/remove pops.

Example: 

```
while (datasets.length > 0) {
  datasets.pop();
}
```

This is just a suggestion though, as I don't know the code base enough to know if there is a reason to hardcode the 2 pops.